backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/CLIParser.java

```diff
@@ -138,9 +113,20 @@ public LicenseInfoParsingResult getLicenseInfo(Attachment attachment) throws TEx
         for (int i = 0; i < nodes.getLength(); i++){
             licenseNamesWithTexts.add(
                     new LicenseNameWithText()
-                    .setLicenseText(findNamedSubelement(nodes.item(i), LICENSE_CONTENT_ELEMENT_NAME).map(Node::getTextContent).map(String::trim).orElse(null))
-                    .setAcknowledgements(findNamedSubelement(nodes.item(i), LICENSE_ACKNOWLEDGEMENTS_ELEMENT_NAME).map(Node::getTextContent).map(String::trim).orElse(null))
-                    .setLicenseName(Optional.ofNullable(nodes.item(i).getAttributes().getNamedItem(LICENSENAME_ATTRIBUTE_NAME))
+                            .setLicenseText(findNamedSubelement(nodes.item(i), LICENSE_CONTENT_ELEMENT_NAME)
+                                    .map(Node::getTextContent)
```

Comment:
It might be good to move this to a function (e.g. `normalizeEscapedXHtml`). Especially since the lines below are doing the same thing.